deploifai/utilities/local_config.py

```diff
@@ -96,6 +94,9 @@ def read_config_file() -> configparser.ConfigParser:
     Read the config file in the existing .deploifai/local.cfg file
     :return: A ConfigParser that contains all the configs in the config file
     """
+    if config_file_path is None:
+        return None
```

Comment:
not sure about this, this is just some error handling I thought of

basically assuming user has not created a project, user does something like `deploifai auth login` or `deploifai --help`


based on my understanding, context obj will look for local config file already, and if no projects created, the global variable `config_file_path` in `local_config.py` should be None

Reply:
nvm if project not created, it won't be `None`, we raise an error instead

but I think the flow should not be like this right?

Reply:
Logging in before creating a project is obviously necessary.

Take a step back a bit, if the config file can be found, we load it into the context as expected.

But if the local config file is not found, I think setting the `context.local_config` to `None` is ok. But there may be some code changes necessary when the `context.local_config` is checked to see if a project id is present. We should just check if `context.local_config` is None instead of doing `if 'id' in context.local_config['Project']`.

Reply:
should we check still for id?